doc/changelog/10-standard-library/12162-bool-leb.rst

```diff
@@ -0,0 +1,4 @@
+- **Changed:**
+  ``Bool.leb`` is renamed into ``Bool.le``. This definition is made local to avoid conflicts with ``Nat.le``. As a consequence, previous calls to ``leb`` based on importing ``Bool`` should now be qualified into ``Bool.le`` even if ``Bool`` is imported.
```

Comment:
This could also be "**Deprecated:** `Bool.leb` in favor of `Bool.le`". With the rest virtually unchanged.